Error installing Neovim

I'm getting an error, here's the tail:

```
...
[ 92%] Built target jemalloc
Warning: Failed searching manifest: Failed extracting manifest file

Error: Failed unpacking rock file: /tmp/luarocks_luarocks-rock-mpack-1.0.6-0-6995/mpack-1.0.6-0.src.rock
make[3]: *** [usr/lib/luarocks/rocks/mpack] Error 1
make[3]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make[2]: *** [CMakeFiles/mpack.dir/all] Error 2
make[2]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make: *** [deps] Error 2
neovim failed to install: `make -j4` exited with 2

┌─[20:36:09/localhost/chronos/~/src]
└─╼ 
```

Hey hey, gave it a shot, but this time it failed like this:

```
[ 87%] No build step for 'luarocks'
[ 88%] Performing install step for 'luarocks'
make[4]: Entering directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/build/src/luarocks'
Warning: variable CFLAGS was not passed in build_variables
make[5]: Entering directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/build/src/luarocks'
make[5]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
make[5]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/build/src/luarocks'
/usr/bin/install -c -m 755 lib/libjemalloc.a /usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/usr/lib
/usr/bin/install -c -m 755 lib/libjemalloc_pic.a /usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/usr/lib
make[4]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/build/src/jemalloc'
[ 89%] Completed 'jemalloc'
make[3]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
[ 89%] Built target jemalloc
Updating manifest for /usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/usr/lib/luarocks/rocks
No existing manifest. Attempting to rebuild...
luarocks scm-1 is now built and installed in /usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/usr (license: MIT/X11)

make[4]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps/build/src/luarocks'
[ 90%] Completed 'luarocks'
make[3]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
[ 90%] Built target luarocks
make[3]: Entering directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
Scanning dependencies of target mpack
make[3]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make[3]: Entering directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
[ 92%] Generating usr/lib/luarocks/rocks/mpack
Warning: Failed searching manifest: Failed extracting manifest file

Error: Failed unpacking rock file: /tmp/luarocks_luarocks-rock-mpack-1.0.6-0-582/mpack-1.0.6-0.src.rock
make[3]: *** [usr/lib/luarocks/rocks/mpack] Error 1
make[3]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make[2]: *** [CMakeFiles/mpack.dir/all] Error 2
make[2]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/usr/local/tmp/crew/v0.1.7.tar.gz.dir/neovim-0.1.7/.deps'
make: *** [deps] Error 2
neovim failed to install: `make -j4` exited with 2
```

--- a/packages/neovim.rb
+++ b/packages/neovim.rb
@@ -11,6 +11,7 @@ class Neovim < Package
   depends_on 'cmake'
   depends_on 'gcc'
   depends_on 'pkgconfig'
+  depends_on 'unzip'
 
   def self.build
     system "make"

